Templates/local_openshift/hogarama-fluentd.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: hogarama-fluentd
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: fluentd-onbuild
+      template: hogarama-fluentd
+    name: fluentd-onbuild
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+    - annotations:
+        openshift.io/generated-by: OpenShiftWebConsole
+        openshift.io/imported-from: fluent/fluentd:edge-debian
+      from:
+        kind: DockerImage
+        name: fluent/fluentd:edge-debian
+      generation: 2
+      importPolicy: {}
+      name: edge-debian
+      referencePolicy:
+        type: Source
+  status:
+    dockerImageRepository: ""
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: fluentd
+      template: hogarama-fluentd
+    name: fluentd
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    creationTimestamp: null
+    labels:
+      app: fluentd
+      template: hogarama-fluentd
+    name: fluentd-log-claim
+  spec:
+    accessModes:
+    - ReadWriteMany
+    resources:
+      requests:
+        storage: 1Gi
+  status: {}
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: fluentd
+      template: hogarama-fluentd
+    name: fluentd
+  spec:
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: fluentd:latest
+    postCommit: {}
+    resources: {}
+    runPolicy: Serial
+    source:
+      contextDir: Fluentd
+      git:
+        ref: ${BRANCH}
+        uri: https://github.com/Gepardec/Hogarama/
+      type: Git
+    strategy:
+      type: Docker
+      dockerStrategy:
+        from:
+          kind: ImageStreamTag
+          name: 'fluentd-onbuild:edge-debian'
+    triggers:
+    - github:
+        secret: 4m9EY9lErQRrs8LJjudP
+      type: GitHub
+    - generic:
+        secret: A2YAGDWxu2zgCfjIyo25
+      type: Generic
+    - type: ConfigChange
+    - imageChange: {}
+      type: ImageChange
+  status:
+    lastVersion: 0
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
```

Comment:
probably not the annotation we want to have when the creation is scripted from this template file